server/data_access_layer/models/localdb/localdb.model.js

```diff
@@ -311,6 +263,144 @@ module.exports = (localdb, Sequelize) => {
+            field: 'may',
+            type: Sequelize.FLOAT,
+            defaultValue: 0
+        },
+        june: {
+            field: 'june',
+            type: Sequelize.FLOAT,
+            defaultValue: 0
+        },
+        july: {
+            field: 'july',
+            type: Sequelize.FLOAT,
+            defaultValue: 0
+        },
+        august: {
+            field: 'august',
+            type: Sequelize.FLOAT,
+            defaultValue: 0
+        },
+        september: {
+            field: 'september',
+            type: Sequelize.FLOAT,
+            defaultValue: 0
+        },
+        october: {
+            field: 'october',
+            type: Sequelize.FLOAT,
+            defaultValue: 0
+        },
+        november: {
+            field: 'november',
+            type: Sequelize.FLOAT,
+            defaultValue: 0
+        },
+        december: {
+            field: 'december',
+            type: Sequelize.FLOAT,
+            defaultValue: 0
+        },
+        total: {
+            field: 'total',
+            type: Sequelize.FLOAT,
+            defaultValue: 0
+        }
+    })
+
+    BillingNumbers.belongsTo(Recipients, {
+        foreignKey: 'employee_id',
+        targetKey: 'employeeId',
+    })
+
+    BillingNumbers.hasOne(BillingNumbers, {
+        foreignKey: {
+            name: 'objectivesId',
+            allowNull: true
+        }
+    })
+
+    const PerformanceReport = localdb.define("performance_reports", {
+        performanceReportId: {
+            field: 'performance_report_id',
+            type: Sequelize.UUID,
+            defaultValue: Sequelize.UUIDV4,
+            primaryKey: true
+        },
+        name: {
+            field: 'performance_report_name',
+            type: Sequelize.STRING
+        },
+        projectedBonus: {
+            field: 'projected_bonus',
+            type: Sequelize.STRING,
+            defaultValue: null
+        }
+    });
+
+    PerformanceReport.belongsTo(User, {
+        foreignKey: {
+            name: 'user_user_id',
+            allowNull: true,
+        }
+    });
+
+    PerformanceReport.belongsTo(ReportType, {
+        foreignKey: {
+            name: 'report_type_id',
+            allowNull: false
+        }
+    })
+
+    PerformanceReport.belongsTo(BillingNumbers, {
+        foreignKey: {
+            name: 'billing_actual_numbers',
+            allowNull: false
+        }
+    })
+
+    PerformanceReport.belongsTo(BillingNumbers, {
+        foreignKey: {
+            name: 'billing_obj_numbers',
+            allowNull: false
+        }
+    })
+
+    PerformanceReport.belongsTo(Recipients, {
+        foreignKey: {
+            name: 'recipient_id',
+            allowNull: false
+        }
+    })
+
 
-    return [User, ChartReport, ChartReportData, PerformanceReport, ReportType, Recipients, ReportTypeRecipients];
+    return [User, ChartReport, ChartReportData, ReportType, Recipients, ReportTypeRecipients, BillingNumbers, PerformanceReport];
```

Comment:
This is the only part I am not sure why you didn't created a model for it, unless for the connection with the foreign key, this is the only way to do it.

Reply:
This file actually needs to be deleted because I separated the models into different files. There is a model for Billing numbers.